### ISSUE TITLE
[Bug]: "FOV Effects" slider within accessibility tab does not work with PuzzlesLib installed

### ISSUE THREAD

**Issue by Arbuckle871** [Fuzss/puzzleslib#153]
### Mod Loader (Required)

Fabric

### Minecraft Version(s) (Required)

1.20.4

### Mod Version(s) (Required)

v20.4.52

### Other Mods Involved (Required)

Yes

### Notes (Required)

When I have PuzzlesLib installed for 1.20.4, my 'FOV Effects' slider in the Accessibility tab of settings does not work no matter the setting, it acts as if the slider is at 100% even if its at 0%. If it is turned all the way to 'OFF' it finally does something by disabling the effect overall. Strange issue to have with this mod and would absolutely love some help!

### latest.log (Required)

https://gist.github.com/Arbuckle871/fcc22d107e090f96f669296a1fa7dcaa

**Comment by Fuzss** [comment]
1.20.4 is not supported unfortunately.
